picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
```

Comment:
Huh. I wonder why this is like this, when it's `'artistsort': 'WM/ArtistSortOrder',` further up for `artistsort`. Maybe `'artists': 'WM/Artists',` would have been better. Does mutagen use all-uppercase? :/